### DISCUSSION TITLE
DXF to JPG image color fill

### DISCUSSION THREAD

**Discussion by mengyangz86** [mozman/ezdxf#891] [Q&A]
Hello, I want to do the following transformation of the image in the DXF file, that is, the color of the fill after converting the JPG is different, how to do this
![MC7F1762](https://github.com/mozman/ezdxf/assets/56494809/9d7f44a0-d6fa-487e-93d2-37a6c65aa7ba)
![1762](https://github.com/mozman/ezdxf/assets/56494809/ed76b56e-d43f-4001-b6fe-122a119c50dc)

**Comment by mbway** [comment]
This is completely out of scope for ezdxf. Once you have an image you are doing image processing not working with DXF.
Try using floodFill() from [opencv](https://pypi.org/project/opencv-python/) or [scikit-image](https://scikit-image.org/docs/stable/auto_examples/segmentation/plot_floodfill.html)
